Lib/multiprocessing/pool.py

```diff
@@ -480,7 +486,7 @@ def _handle_results(outqueue, get, cache):
                 util.debug('result handler got EOFError/OSError -- exiting')
                 return
 
-            if thread._state:
+            if thread._state != "RUN":
```

Comment:
@vstinner, @ulfalizer mentioned on 2b417fb that this may have meant to be `RUN`, not `"RUN"`

Reply:
Oops! Fixed by PR #11178.